build.sbt

```diff
@@ -1,29 +1,53 @@
+import sbtrelease.ReleaseStateTransformations.{checkSnapshotDependencies, commitNextVersion, commitReleaseVersion, inquireVersions, pushChanges, runClean, runTest, setNextVersion, setReleaseVersion, tagRelease}
+
 name := "configuration"
 
 organization := "com.gu"
 
+licenses := Seq("APL2" -> url("http://www.apache.org/licenses/LICENSE-2.0.txt"))
+homepage := Some(url("https://github.com/guardian/guardian-configuration"))
+scmInfo := Some(ScmInfo(
+  url("https://github.com/guardian/guardian-configuration"),
+  "scm:git@github.com:guardian/guardian-configuration.git"
+))
+developers := List(
+  Developer(id="guardian", name="Guardian", email="", url=url("https://github.com/guardian")),
+)
+releasePublishArtifactsAction := PgpKeys.publishSigned.value
+releaseProcess := Seq[ReleaseStep](
+  checkSnapshotDependencies,
+  inquireVersions,
+  runClean,
+  releaseStepCommandAndRemaining("+test"),
+  setReleaseVersion,
+  commitReleaseVersion,
+  tagRelease,
+  releaseStepCommandAndRemaining("+publishSigned"),
+  setNextVersion,
+  commitNextVersion,
+  releaseStepCommand("sonatypeReleaseAll"),
+  pushChanges
+)
```

Comment:
I have only tested publishing a snapshot. I'm not sure how to validate these release settings are right without actually attempting a release. 
I copied this from [another project](https://github.com/guardian/identity/blob/main/build.sbt#L44-L57) but I have not tested it here